etc/zfsbootmenu/mkinitcpio.conf

```diff
@@ -94,6 +94,11 @@ HOOKS=(base udev autodetect modconf block filesystems keyboard zfsbootmenu)
 # environment is about to be launched. Any writable ZFS pools will have been
 # exported, but read-only pools will generally still be available.
 #
+# 'zfsbootmenu_load_key' hooks are run immediately before attempting to load an
```

Comment:
This text needs to be before the text for `zfsbootmenu_teardown`.

Reply:
done